src/Serialize.Linq.NetStandard16/Serialize.Linq.NetStandard16.csproj

```diff
@@ -1,18 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netstandard1.6</TargetFramework>
     <RootNamespace>Serialize.Linq</RootNamespace>
-    <Version>1.4.0</Version>
+    <Version>1.4.0$(VersionSuffix)</Version>
     <Company />
-    <Product>Serialize.Linq.NetCore11</Product>
+    <Product>Serialize.Linq.NetStandard16</Product>
```

Comment:
I think `Serialize.Linq.NetStandard` would be a better name, leaving the version info out